frontend/src/Modules/Career/Components/Overview/Overview.js

```diff
@@ -1,94 +1,163 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Container from '@material-ui/core/Container';
-import { makeStyles } from '@material-ui/core/styles';
 import Timeline from '@material-ui/lab/Timeline';
 import TimelineItem from '@material-ui/lab/TimelineItem';
 import TimelineSeparator from '@material-ui/lab/TimelineSeparator';
 import TimelineConnector from '@material-ui/lab/TimelineConnector';
 import TimelineContent from '@material-ui/lab/TimelineContent';
-import TimelineOppositeContent from '@material-ui/lab/TimelineOppositeContent';
+import CareerService from '../../Shared/CareerService'
 import TimelineDot from '@material-ui/lab/TimelineDot';
-import Paper from '@material-ui/core/Paper';
-import Typography from '@material-ui/core/Typography';
-
-const useStyles = makeStyles((theme) => ({
-    paper: {
-        padding: '6px 16px',
-    },
-    secondaryTail: {
-        backgroundColor: theme.palette.secondary.main,
-    },
-}));
+import { Paper, Button, Typography } from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import FormCareer from '../../Detail/FormCareer';
+import _FormCareer from '../../Detail/FormCareer';
+import TimelineOppositeContent from '@material-ui/lab/TimelineOppositeContent';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import TodayIcon from '@material-ui/icons/Today';
+import Slide from '@material-ui/core/Slide';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import InfiniteScroll from 'react-infinite-scroll-component';
+class Overview extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            anchorEl: false,
+            open: false,
+            careers: [],
+            sliceNumber: 2,
+            userId: '60d005eadbec139b90add9a9',
```

Comment:
is this fake ID?

Reply:
vâng chỗ này e hardcoded cái userID để sử dụng tạm
merge xong e chuyển sang bên user để gọi cái component này rồi truyền ID sau